Sources/StreamChat/ChatClient.swift

```diff
@@ -456,7 +456,6 @@ public class ChatClient {
     /// Disconnects the chat client from the chat servers. No further updates from the servers
     /// are received.
     public func disconnect(completion: @escaping () -> Void) {
-        connectionRecoveryHandler?.stop()
```

Comment:
Seems to be safe because recovery handler checks for current connection state and does not attempt to reconnect  if the state is `disconnected` with `userInitiated`. 🤔 


```
var canReconnectFromOffline: Bool {
        …
        switch webSocketClient.connectionState {
        case .disconnected(let source) where source == .userInitiated:
            return false
```

Reply:
Yes exactly the reconnection handler although alive, it won't do nothing